governance.md

```diff
@@ -11,6 +11,55 @@ Contributions from the community are are welcome and accepted via pull requests.
 | member | Active contributor in the community. | Sponsored by 2 reviewers.  Multiple contributions to the project. | CSI GitHub org member. |
 | reviewer | Review contribution from other members. | History of review and authorship. | OWNERS file reviewer entry. |
 | approver | Approve contributions for merge. | Core group of CO representatives. | OWNERS file approver entry. |
-| janitor | Org and repo maintenance. | Minimal subset of approvers required for maintanence. | GitHub repo write access. |
+| janitor | Org and repo maintenance. | Minimal subset of approvers required for maintenance. | GitHub repo write access. |
 
 While contributions are accepted from the community at large, to maintain impartiality and the benefit of end users, approvers for CSI are limited to just this core group of CO representatives.
+
+## Voting
+
+Voting on a motion must happen on the mailing list.
+
+A motion can be one of the following:
+* Package releases
+* Procedural issues
+
+### Voting Window
+
+Voting on each motion must specifiy a time window when voters can vote.
+Any vote made outside the time window will not be considered.
+Votes should be permitted to run for at least 72 hours to provide an opportunity for all concerned persons to participate regardless of their geographic locations.
```

Comment:
"Votes must be permitted to run for at least 72 hours OR until all votes have been cast"

Also maybe "Votes must not run more than X days" (1 week? 2 weeks?)...